src/cgame/cg_snapshot.cpp

```diff
@@ -124,7 +124,7 @@ void CG_SetInitialSnapshot( snapshot_t *snap )
 	}
 
 	// Need this check because the initial weapon for spec isn't always WP_NONE
-	if ( CG_MyTeam() == TEAM_NONE )
```

Comment:
I checked what CG_MyTeam() does, and it checks `cg.predictedPlayerState.persistant[PERS_TEAM]`.
Why the change? (I don't say this is wrong, I focused my efforts on sgame so far)

BTW we should change `G_Team( gentity_t* )` so that it becomes `BG_Team( playerState_t const& )` to avoid that kind of things... I thought this already existed, but after checking, nope, it does not.

Reply:
It doesn't work with predictedPlayerState - it's all zero the first time this executes

Reply:
I see. Thanks for explanations.